about.css

```diff
@@ -1,3 +1,9 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] As suggested by the last reviewer, kindly add two more cards to have the same number as specified in the design given.
-The idea is to have you replicate the same design. Kindly add the remaining cards 👍 
- [Optional] The color of the cards is not the same as yours. To have a better design and have a replica that fits perfectly, kindly consider adding the right color which is grey, and The hover on the cards which the last reviewer missed 👍 

## Your work:
![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/50990925/e58703d1-2514-45bf-b15b-875a9e8c7c87)

## Design:
![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/50990925/16d64320-24ad-48ae-a20a-6701b662191b)